<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaCertificatesPage</class>
 <widget class="QWidget" name="CaCertificatesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>562</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <column>
      <property name="text">
       <string>Organization / Common Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Organizational Unit</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="displaySelection">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Display...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="disableSelection">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enableSelection">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeSelection">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
